pymatgen/io/lobster/outputs.py

```diff
@@ -1116,62 +1116,84 @@ class Fatband:
             The indices of the array are [band_index, kpoint_index].
             The dict is then built the following way: {"string of element": "string of orbital as read in
             from FATBAND file"}. If the band structure is not spin polarized, we only store one data set under Spin.up.
-        structure (Structure): Structure read in from vasprun.xml.
+        structure (Structure): Structure read in from structure_file or Structure object.
     """
 
-    def __init__(self, filenames=".", vasprun="vasprun.xml", Kpointsfile="KPOINTS"):
+    def __init__(
+        self,
+        filenames: str | list = ".",
+        kpoints_file: str = "KPOINTS",
+        structure_file: str | None = "POSCAR.lobster",
```

Comment:
following Python's "there should be one obvious way to do something" i'd advocate for removing the `structure_file` keyword. afaict, it's only saving the user 1 line

```py
Structure.from_file(structure_file)
```